### DISCUSSION TITLE
Placement of sorting handles in data tables

### DISCUSSION THREAD

**Discussion by ElectricCookie** [emdgroup-liquid/liquid#260] [General]
Hi guys 👋 ,

while mocking up a data table I just noticed that in the figma designs the sorting handles ↕️  for the data table are aligned to the right ➡️  side of a table cell:

![image](https://user-images.githubusercontent.com/18628516/151162597-320ddc5d-81c6-45c8-9d2e-6ece692e57d3.png)
Source: Figma - Themes page

Since the data tables don't have vertical dividers I think the relationship between text and sorting handle gets lost in my opinion ( the sorting handle for the respective column is closer to the header of the next column). 

 Regards 🙌

**Comment by NicolasBlock91** [comment]
Hey, in some cases in the past the alignment to the right lead to a more structured table design. Especially between the sorting handle and the edit/ action icons in the rows underneath.
<img width="44" alt="1" src="https://user-images.githubusercontent.com/89650126/151752909-5d45cbd3-0ec0-4b8f-a39d-8c89e90c0085.png">

But I understand exactly what you mean @ElectricCookie, so there is the possibility of changing the default behaviour of the text box from "fill container" to "hug contents". This change will then lead to your expected behaviour.
<img width="244" alt="3" src="https://user-images.githubusercontent.com/89650126/151752874-809f4b75-b3f1-4915-9f29-62a7e64acfa7.png">
<img width="370" alt="2" src="https://user-images.githubusercontent.com/89650126/151752877-4270fa11-970b-4c7f-bf1e-fee2440bf9af.png">
<img width="617" alt="Screenshot 2022-01-31 at 08 21 55" src="https://user-images.githubusercontent.com/89650126/151753666-ef29aaad-dc67-40fc-a642-485c54eae328.png">

Best Nicolas

**Reply by ElectricCookie** [reply]
That's a nice option!

I must say the first option is optically more pleasing, but I think "hug contents" has better usability. 

Thanks! 

👍
